### PR TITLE
Add databooks hook

### PR DESCRIPTION
**Pull request by murilo-cunha**

### CODE REVIEW

File: all-repos.yaml [hunk 1]
```diff
@@ -210,3 +210,4 @@
 - https://github.com/adamchainz/pre-commit-oxipng
 - https://github.com/gitguardian/gg-shield
 - https://github.com/JohnnyMorganz/StyLua
+- https://github.com/datarootsio/databooks
```
Comment (asottile) [inline all-repos.yaml:213]: I believe this incorrectly sets [`require_serial: true`](https://github.com/datarootsio/databooks/blob/a7b627b57a53d2562553e099684f5b32a3a6acc1/.pre-commit-hooks.yaml#L8)

Comment (murilo-cunha) [reply all-repos.yaml:213]: Hmm why would this be incorrect? I thought it'd be best to `require_serial` as it'd write files to disk (similar to `black`)

Comment (asottile) [reply all-repos.yaml:213]: that's not why black has `require_serial` -- black has `require_serial` because it does work using multiple processes

Comment (murilo-cunha) [reply all-repos.yaml:213]: I see I see, [I've removed it](https://github.com/datarootsio/databooks/blob/main/.pre-commit-hooks.yaml)! Thanks for the explanation
And nice tool btw 🙃

Comment (asottile) [reply all-repos.yaml:213]: thanks! you'll need to rebase this unfortunately due to conflicts

Comment (murilo-cunha) [reply all-repos.yaml:213]: Done ✅

**Pull request merged**